tsconfig.json

```diff
@@ -73,6 +76,9 @@
     {
       "path": "./packages/cactus-verifier-client/tsconfig.json"
     },
+    {
+      "path": "./packages/cactus-plugin-persistence-ethereum/tsconfig.json"
```

Comment:
@ervvkb This file does not exist on the main branch, nor in your PR diff. Was this added by mistake?

Reply:
Resolving because now I know that the parent PR has this in it's diff.